jekyll/_cci2/docker-layer-caching.md

```diff
@@ -17,20 +17,19 @@ Consider reusing the unchanged layers to significantly reduce image build times.
 
 ``` YAML
 - setup_remote_docker:
-    reusable: true    # default - false
-    exclusive: true   # default - true
+    docker_layer_caching: true # default - true  
 ```
 
-## **`reusable`**
+When `docker_layer_caching` is set to `true`, CircleCI will try to reuse your Remote Docker Environment. That is, every layer you built in a previous job will be accessible in the remote environment and CircleCI will attempt to reuse the previous environment when it is possible. However, in some cases your job may run in a clean environment, even if the configuration specifies `docker_layer_caching: true`.
 
-When `reusable` is set to `true`, CircleCI will try to reuse your Remote Docker Environment. That is, every layer you built in a previous job will be accessible in the remote environment and CircleCI will attempt to reuse the previous environment when it is possible. However, in some cases your job may run in a clean environment, even if the configuration specifies `reusable: true`.
+If you run many parallel jobs for the same project that depend on the same environment, all of them will be provided with a Remote Docker Environment. Docker Layer Caching guarantees jobs to have exclusive Remote Docker Environments that other jobs cannot access. However, not all of the jobs will have cached layers, although this behavior is subject to change in a future update.
 
-If you run many parallel jobs for the same project, and each job requests a reusable environment, all of them will be provided with a Remote Docker Environment. However, not all of the jobs will have cached layers, although this behavior is subject to change in a future update.
+**Note:** Previously the `docker_layer_caching` was called `reusable`. The `reusable` key is deprecated in favor of the `docker_layer_caching` key. 
 
-## **`exclusive`**
+In addition, the `exclusive` option is deprecated in favor of all VMs being treated as exclusive. This indicates that jobs are guaranteed to have an exclusive Remote Docker Environment that other jobs cannot access when using `docker_layer_caching` with `machine`.
 
-The second option (`exclusive`) indicates whether you want to allow parallel jobs to run simultaneously in the same Remote Docker Environment. Jobs for which `exclusive:` is set to `true` are guaranteed to have an exclusive Remote Docker Environment that other jobs cannot access.
-
-If you set `exclusive:` to `false`, the project's parallel jobs will be able to share the same Remote Docker Environment. This method lets you reduce the chances of receiving a Remote Docker Environment without a cache. If you choose this option, ensure that your project can handle concurrent operations in Docker Engine.
+``` YAML
```

Comment:
Here we need to add a section "Docker Layer Caching in `machine` executor". And say that DLC is available not only for Remote Docker, but also for `machine` executor, that it works exactly the same way and enables as in the example below.

Reply:
Also at the beginning of this article, there's a sentence "Because Docker Layer Caching uses the `setup_remote_docker` step, it is not compatible with the `machine` executor." - we need to remote it.

Reply:
Also, before a paragraph "Consider reusing the unchanged layers" we need to add a title "Docker Layer Caching in Remote Docker"

Reply:
fixed, thank you!